src/app/layout.js

```diff
@@ -0,0 +1,22 @@
+import { Inter } from 'next/font/google';
+import './globals.css';
+import './prism-a11y-dark.css';
+import '@/components/Projects/ProjectList.css';
+import FaroContextProvider from '@/context/FaroContextProvider';
+
+const inter = Inter({ subsets: ['latin'] });
+
+export const metadata = {
```

Comment:
On second thought I should lowkey do away with CustomHeader since doing this is the same thing 💀